apiml-extension-loader/src/main/java/org/zowe/apiml/extension/ExtensionsLoader.java

```diff
@@ -34,11 +34,8 @@ public class ExtensionsLoader implements ApplicationListener<ApplicationContextI
 
     @Override
     public void onApplicationEvent(ApplicationContextInitializedEvent event) {
-        if (!(event.getApplicationContext() instanceof BeanDefinitionRegistry)) {
-            log.error("Expected Spring context to be a BeanDefinitionRegistry. Extensions are not loaded");
-        } else {
-            BeanDefinitionRegistry registry = (BeanDefinitionRegistry) event.getApplicationContext();
-            ClassPathBeanDefinitionScanner scanner = new ClassPathBeanDefinitionScanner(registry);
+        if (event.getApplicationContext() instanceof BeanDefinitionRegistry registry) {
+            ClassPathBeanDefinitionScanner scanner = new ClassPathBeanDefinitionScanner(new AnnotationConfigApplicationContext());
```

Comment:
It could've been just a one line change..

Reply:
But the SonarQube was against...